sim/lib/settings/town_settings_sanfrancisco.py

```diff
@@ -83,47 +83,31 @@ def _essential_distribution():
     0])     # 80+
     return ed
 
-
```

Comment:
This part is no longer used in the town generator, so I removed it to prevent confusion in the future.